### DISCUSSION TITLE
raw.githubusercontent.com - not reachable via IPv6 (Fastly CDN)

### DISCUSSION THREAD

**Discussion by presianbg** [community/community#32501] [Other Feature Feedback, Questions, & Ideas]
Hi,

For the past few weeks I have constant problems with reaching raw.githubusercontent.com via my IPv6 connectivity.
The DNS works just fine (tried both CloudFlare - 1.1.1.1 and Google - 8.8.8.8) , the problem is in the Fastly CND which happens to serve Github's content via IPv6! 

Can somebody check if they have the same issue ?

EDIT: Forgot to mention that I have NO problems with other services / websites via IPv6.

Cheers,
PY

**Comment by devarsh-mavani-19** [comment]
@presianbg Did u find any solution?

**Reply by presianbg** [reply]
Nope, I gave up and forced my local DNS resolver to not return `AAAA` answers for ` raw.githubusercontent.com`.
The issue might be in the CND anycast network or something, so it's not something we can do about it.
